docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -7,9 +7,9 @@
 
 = Installing the {ProductName} {WebName}
 
-You can install the {ProductName} ({ProductShortName}) {WebName} as part of the process of installing the {ProductShortName} Operator on the OpenShift Container Platform.
+You can install the {ProductName} ({ProductShortName}) {WebName} on all cloud services and self-managed OpenShift editions.
```

Comment:
```suggestion
You can install the {ProductName} ({ProductShortName}) {WebName} on all OpenShift cloud services and self-managed OpenShift editions.
```

Reply:
@rromannissen, as for "this wording sounds weird", what exactly is incorrect in this case? From what I understand from the initial sentence version is that the MTA UI is installed when the user installs the MTA Operator. Is this wrong?